pkg/version/version.go

```diff
@@ -30,6 +30,14 @@ type Info struct {
 	Components        // The same version as VdbVer but broken down into individual components
 }
 
+type ComparisonResult int
+
+const (
+	CompareEqual   ComparisonResult = iota // CompareEqual represents the state where versions are equal
+	CompareLarger                          // CompareLarger represents the state where the first version is larger
+	CompareSmaller                         // CompareSmaller represents the state where the first version is smaller
```

Comment:
Lets avoid exporting these const values. Can you begin them with lowercase?

Reply:
So we are keeping these changes?

Reply:
changed to lowercase

Reply:
I was okay with the refactor.